src/lib/get-topper-settings.js

```diff
@@ -98,12 +98,21 @@ const usePackageTopper = (content) => {
 	};
 };
 
-const useEditoriallySelectedTopper = (content) => {
+const useEditoriallySelectedTopper = (content, flags) => {
 	let backgroundColour;
+	let myFtButton;
 
 	// Convert old palette colours to new palette colours from Methode
 	if (content.topper.layout === 'full-bleed-offset') {
 		backgroundColour = 'wheat';
+	} else if (content.topper.layout === 'deep-landscape') {
+		if (['paper', 'wheat'].includes(content.topper.backgroundColour)) {
```

Comment:
```suggestion
		if (['paper', 'wheat','pink','auto'].includes(content.topper.backgroundColour)) {
```
They are mapped below as wheat so i think we need to catch them too

Reply:
Suggestion: I'm worried that this code is getting out of control. We need to pay particular attention at the way we write it.
We should avoid having a hardcoded list of colours here cause it could easily get out-of-sync with other colours added or removed.

Similarly to [here](https://github.com/Financial-Times/n-map-content-to-topper/blob/4efb8cab4e473377d7946347ff53bdb503b738b5/src/map-content-to-topper.js#L13) we can have a method that tell us if a background is dark or light.

You should perhaps starting from [here](https://github.com/Financial-Times/n-map-content-to-topper/blob/4efb8cab4e473377d7946347ff53bdb503b738b5/src/map-content-to-topper.js#L8), moving to something like...

```
const hasLightBackground = (backgroundColour) => {
        const lightBackgrounds = ['paper', 'wheat', 'white']; <<< Do we need `auto` here? 
	return lightBackgrounds.indexOf(backgroundColour) > -1;
}


const myFtButtonVariant = (backgroundColour) => {
	return !backgroundColour || hasLightBackground() ? 'standard' : 'monochrome';
};
```
or perhaps you could just relying on the opposite result of hasDarkBacground...
```
const hasLightBackground(backgroundColour) => !hasDarkBackground(backgroundColour)
```
so ideally we would like to arrive to something here like...

```suggestion
		if (hasLightBackground(backgroundColour)) {
```